src/include/pbs_entlim.h

```diff
@@ -63,78 +59,58 @@ enum lim_keytypes {
 };
 
 #define PBS_GENERIC_ENTITY "PBS_GENERIC"
-#define PBS_ALL_ENTITY     "PBS_ALL"
-
-#define ETLIM_INVALIDCHAR	"/[]\";:|<>+,?*"
+#define PBS_ALL_ENTITY	   "PBS_ALL"
```

Comment:
shall we align the values?

Reply:
I did align those values, looks like github diff UI not showing it correctly but if you go to file and see then those values will be aligned.